task2/task2_solution.py

```diff
@@ -0,0 +1,84 @@
+class Journal:
+    student_data = []
```

Comment:
There is no need to initialize your list here, all your initialization is already in `__init__`